tasks.py

```diff
@@ -220,28 +257,44 @@ def buildx(
     cache=False,
     cache_dir="",
     platforms="linux/amd64",
-    tag="networktocode/nautobot-dev-py3.7:local",
+    tag=None,
     target="dev",
     poetry_parallel=False,
 ):
-    """Build Nautobot docker image using the experimental buildx docker functionality (multi-arch capablility)."""
-    print(f"Building Nautobot with Python {context.nautobot.python_ver} for {platforms}...")
-    command = (
-        f"docker buildx build --platform {platforms} -t {tag} --target {target} --load -f ./docker/Dockerfile"
-        f" --build-arg PYTHON_VER={context.nautobot.python_ver}"
-        " ."
-    )
+    """Build Nautobot docker image using the experimental buildx docker functionality (multi-arch capability)."""
+    print(f"Building Nautobot {target} target with Python {context.nautobot.python_ver} for {platforms}...")
+    if tag is None:
+        if target == "dev":
+            tag = f"networktocode/nautobot-dev-py{context.nautobot.python_ver}:local"
+        elif target == "final":
+            tag = f"networktocode/nautobot-py{context.nautobot.python_ver}:local"
+        else:
+            print(f"Not sure what should be the standard tag for target {target}, will not tag.")
+    command_tokens = [
+        "docker buildx build .",
+        f"--platform {platforms}",
+        f"--target {target}",
+        "--load",
+        "-f ./docker/Dockerfile",
+        f"--build-arg PYTHON_VER={context.nautobot.python_ver}",
+    ]
+    if tag is not None:
+        command_tokens.append(f"-t {tag}")
     if not cache:
-        command += " --no-cache"
+        command_tokens.append("--no-cache")
     else:
-        command += (
-            f" --cache-to type=local,dest={cache_dir}/{context.nautobot.python_ver}"
-            f" --cache-from type=local,src={cache_dir}/{context.nautobot.python_ver}"
-        )
+        command_tokens += [
+            f"--cache-to type=local,dest={cache_dir}/{context.nautobot.python_ver}",
+            f"--cache-from type=local,src={cache_dir}/{context.nautobot.python_ver}",
+        ]
     if poetry_parallel:
-        command += " --build-arg POETRY_PARALLEL=true"
+        command_tokens.append("--build-arg POETRY_PARALLEL=true")
 
-    context.run(command, env={"PYTHON_VER": context.nautobot.python_ver})
+    command = " ".join(command_tokens)
```

Comment:
Wondering if we might to use `shlex.join()` here instead, but no big deal.

Reply:
I always forget about the existence of `shlex`. Seems like it wouldn't hurt. I'll give it a try.

Reply:
`shlex.join` requires Python 3.8 or later, sadly.